auth/utils/file-reading-helpers.browser.ts

```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright 2021 IBM Corp. All Rights Reserved.
```

Comment:
This is a little annoying but I wonder if we need to be making sure these dates are accurate. As in, the date should reflect when the file was created and the last time it was edited. Not sure how important this is but Mike always said it was. He even wrote this script to automate it - https://github.ibm.com/mkistler/lint-copyright-dates

Might be worth checking out to see if it's a simple thing to do